datanode/gateway/graphql/schema.graphql

```diff
@@ -677,6 +677,18 @@ type Query {
   "Get statistics about the Vega node"
   statistics: Statistics!
 
+  "List markets in a succession chain"
```

Comment:
On second thought, how about the "succession line" or another phrase that doesn't use the word chain?